2015-12-12 18:32:31,616 INFO     [pytan.handler] PyTan User config file successfully loaded: /Users/jolsen/.pytan_config.json 
2015-12-12 18:32:31,617 DEBUG    [pytan.handler] Using value for argument 'username' supplied via pytan user config
2015-12-12 18:32:31,617 DEBUG    [pytan.handler] Using value for argument 'password' supplied via pytan user config
2015-12-12 18:32:31,617 DEBUG    [pytan.handler] Using value for argument 'session_id' supplied via pytan defaults
2015-12-12 18:32:31,617 DEBUG    [pytan.handler] Using value for argument 'host' supplied via pytan user config
2015-12-12 18:32:31,617 DEBUG    [pytan.handler] Using value for argument 'port' supplied via pytan defaults
2015-12-12 18:32:31,617 DEBUG    [pytan.handler] Using value for argument 'loglevel' supplied via class argument
2015-12-12 18:32:31,617 DEBUG    [pytan.handler] Using value for argument 'loggmt' supplied via pytan defaults
2015-12-12 18:32:31,617 DEBUG    [pytan.handler] Using value for argument 'logconsole_formatter' supplied via pytan defaults
2015-12-12 18:32:31,617 DEBUG    [pytan.handler] Using value for argument 'logconsole_enable' supplied via pytan defaults
2015-12-12 18:32:31,617 DEBUG    [pytan.handler] Using value for argument 'logfile_enable' supplied via pytan defaults
2015-12-12 18:32:31,617 DEBUG    [pytan.handler] Using value for argument 'logfile_output' supplied via pytan defaults
2015-12-12 18:32:31,617 DEBUG    [pytan.handler] Using value for argument 'logfile_formatter' supplied via pytan defaults
2015-12-12 18:32:31,617 DEBUG    [pytan.handler] Using value for argument 'pytan_user_config' supplied via pytan defaults
2015-12-12 18:32:31,816 DEBUG    [pytan.utils.network] Port test to 10.0.1.240:443 SUCCESS
2015-12-12 18:32:31,817 DEBUG    [pytan.session.auth] Using Username/Password for authentication headers
2015-12-12 18:32:31,817 DEBUG    [pytan.session.http] HTTP request: GET to https://10.0.1.240:443/auth
2015-12-12 18:32:31,817 DEBUG    [pytan.session.http] HTTP request: headers: {'username': 'QWRtaW5pc3RyYXRvcg==', 'Connection': 'keep-alive', 'Accept-Encoding': 'gzip, deflate', 'password': '**PASSWORD**', 'Accept': '*/*', 'User-Agent': 'python-requests/2.7.0 CPython/2.7.10 Darwin/14.5.0'}
2015-12-12 18:32:31,829 INFO     [pytan.utils.external.requests.packages.urllib3.connectionpool] Starting new HTTPS connection (1): 10.0.1.240
2015-12-12 18:32:31,843 DEBUG    [pytan.utils.external.requests.packages.urllib3.connectionpool] "GET /auth HTTP/1.1" 200 134
2015-12-12 18:32:31,844 DEBUG    [pytan.session.http] HTTP response: from 'https://10.0.1.240:443/auth' len:134, status:200 OK, body type: <type 'unicode'>
2015-12-12 18:32:31,844 DEBUG    [pytan.session.http] HTTP response: headers: {'x-frame-options': 'SAMEORIGIN', 'content-type': 'text/plain; charset=us-ascii', 'connection': 'keep-alive', 'content-length': '134'}
2015-12-12 18:32:31,844 DEBUG    [pytan.session.body] HTTP response: body:
1-398-1a341619970cac26a7046961dc73f83c53c916ba08e4d875d7105f60f14b6072add59a028e4f79c782475d08af76a46d913bdd4daa7308a139a4aa02d1008bc7
2015-12-12 18:32:31,844 DEBUG    [pytan.session.auth] Session ID updated to: 1-398-1a341619970cac26a7046961dc73f83c53c916ba08e4d875d7105f60f14b6072add59a028e4f79c782475d08af76a46d913bdd4daa7308a139a4aa02d1008bc7
2015-12-12 18:32:31,844 DEBUG    [pytan.session.auth] Successfully authenticated and received a non-persistent (up to 5 minutes) session id using username/password
2015-12-12 19:45:08,888 INFO     [pytan.handler] PyTan User config file successfully loaded: /Users/jolsen/.pytan_config.json 
2015-12-12 19:45:08,890 DEBUG    [pytan.handler] Using value for argument 'username' supplied via pytan user config
2015-12-12 19:45:08,890 DEBUG    [pytan.handler] Using value for argument 'password' supplied via pytan user config
2015-12-12 19:45:08,890 DEBUG    [pytan.handler] Using value for argument 'session_id' supplied via pytan defaults
2015-12-12 19:45:08,890 DEBUG    [pytan.handler] Using value for argument 'host' supplied via pytan user config
2015-12-12 19:45:08,890 DEBUG    [pytan.handler] Using value for argument 'port' supplied via pytan defaults
2015-12-12 19:45:08,890 DEBUG    [pytan.handler] Using value for argument 'loglevel' supplied via class argument
2015-12-12 19:45:08,890 DEBUG    [pytan.handler] Using value for argument 'loggmt' supplied via pytan defaults
2015-12-12 19:45:08,890 DEBUG    [pytan.handler] Using value for argument 'logconsole_formatter' supplied via pytan defaults
2015-12-12 19:45:08,890 DEBUG    [pytan.handler] Using value for argument 'logconsole_enable' supplied via pytan defaults
2015-12-12 19:45:08,890 DEBUG    [pytan.handler] Using value for argument 'logfile_enable' supplied via pytan defaults
2015-12-12 19:45:08,890 DEBUG    [pytan.handler] Using value for argument 'logfile_output' supplied via pytan defaults
2015-12-12 19:45:08,890 DEBUG    [pytan.handler] Using value for argument 'logfile_formatter' supplied via pytan defaults
2015-12-12 19:45:08,891 DEBUG    [pytan.handler] Using value for argument 'pytan_user_config' supplied via pytan defaults
2015-12-12 19:45:09,208 DEBUG    [pytan.utils.network] Port test to 10.0.1.240:443 SUCCESS
2015-12-12 19:45:09,208 DEBUG    [pytan.session.auth] Using Username/Password for authentication headers
2015-12-12 19:45:09,209 DEBUG    [pytan.session.http] HTTP request: GET to https://10.0.1.240:443/auth
2015-12-12 19:45:09,209 DEBUG    [pytan.session.http] HTTP request: headers: {'username': 'QWRtaW5pc3RyYXRvcg==', 'Connection': 'keep-alive', 'Accept-Encoding': 'gzip, deflate', 'password': '**PASSWORD**', 'Accept': '*/*', 'User-Agent': 'python-requests/2.7.0 CPython/2.7.10 Darwin/14.5.0'}
2015-12-12 19:45:09,221 INFO     [pytan.utils.external.requests.packages.urllib3.connectionpool] Starting new HTTPS connection (1): 10.0.1.240
2015-12-12 19:45:09,237 DEBUG    [pytan.utils.external.requests.packages.urllib3.connectionpool] "GET /auth HTTP/1.1" 200 134
2015-12-12 19:45:09,238 DEBUG    [pytan.session.http] HTTP response: from 'https://10.0.1.240:443/auth' len:134, status:200 OK, body type: <type 'unicode'>
2015-12-12 19:45:09,238 DEBUG    [pytan.session.http] HTTP response: headers: {'x-frame-options': 'SAMEORIGIN', 'content-type': 'text/plain; charset=us-ascii', 'connection': 'keep-alive', 'content-length': '134'}
2015-12-12 19:45:09,239 DEBUG    [pytan.session.body] HTTP response: body:
1-399-06a16839a4014ce3a22da28ae76cdf6ee10b5e638160bedeaa8c73dd1b1033f025e9004b081f5df628d8db9314198ff552f154c8d2156e24683fe71ca51ef850
2015-12-12 19:45:09,239 DEBUG    [pytan.session.auth] Session ID updated to: 1-399-06a16839a4014ce3a22da28ae76cdf6ee10b5e638160bedeaa8c73dd1b1033f025e9004b081f5df628d8db9314198ff552f154c8d2156e24683fe71ca51ef850
2015-12-12 19:45:09,239 DEBUG    [pytan.session.auth] Successfully authenticated and received a non-persistent (up to 5 minutes) session id using username/password
2015-12-12 22:15:24,381 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.network' to DEBUG
2015-12-12 22:15:24,382 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.extractors' to DEBUG
2015-12-12 22:15:24,382 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.mappers' to DEBUG
2015-12-12 22:15:24,382 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.parsers' to DEBUG
2015-12-12 22:15:24,382 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.pretty' to DEBUG
2015-12-12 22:15:24,382 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.tanium_obj' to DEBUG
2015-12-12 22:15:24,383 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.xml_clean' to DEBUG
2015-12-12 22:15:24,383 DEBUG    [pytan.utils.log] set_levels: setting logger 'root' to DEBUG
2015-12-12 22:15:51,601 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.network' to DEBUG
2015-12-12 22:15:51,601 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.extractors' to DEBUG
2015-12-12 22:15:51,601 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.mappers' to DEBUG
2015-12-12 22:15:51,601 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.parsers' to DEBUG
2015-12-12 22:15:51,601 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.pretty' to DEBUG
2015-12-12 22:15:51,601 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.tanium_obj' to DEBUG
2015-12-12 22:15:51,601 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.xml_clean' to DEBUG
2015-12-12 22:15:51,601 DEBUG    [pytan.utils.log] set_levels: setting logger 'root' to DEBUG
2015-12-12 22:17:15,144 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.network' to DEBUG
2015-12-12 22:17:15,144 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.extractors' to DEBUG
2015-12-12 22:17:15,144 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.mappers' to DEBUG
2015-12-12 22:17:15,144 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.parsers' to DEBUG
2015-12-12 22:17:15,144 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.pretty' to DEBUG
2015-12-12 22:17:15,144 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.tanium_obj' to DEBUG
2015-12-12 22:17:15,144 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.xml_clean' to DEBUG
2015-12-12 22:17:15,144 DEBUG    [pytan.utils.log] set_levels: setting logger 'root' to DEBUG
2015-12-12 22:24:24,505 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.network' to DEBUG
2015-12-12 22:24:24,505 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.extractors' to DEBUG
2015-12-12 22:24:24,505 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.mappers' to DEBUG
2015-12-12 22:24:24,506 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.parsers' to DEBUG
2015-12-12 22:24:24,506 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.pretty' to DEBUG
2015-12-12 22:24:24,506 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.tanium_obj' to DEBUG
2015-12-12 22:24:24,506 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.xml_clean' to DEBUG
2015-12-12 22:24:24,506 DEBUG    [pytan.utils.log] set_levels: setting logger 'root' to DEBUG
2015-12-12 22:24:24,507 DEBUG    [pytan.utils.network] Port test to 10.0.1.240:443 SUCCESS
2015-12-12 22:24:24,508 DEBUG    [pytan.session.auth] Using Username/Password for authentication headers
2015-12-12 22:24:24,508 DEBUG    [pytan.session.http] HTTP request: GET to https://10.0.1.240:443/auth
2015-12-12 22:24:24,508 DEBUG    [pytan.session.http] HTTP request: headers: {'username': 'QWRtaW5pc3RyYXRvcg==', 'Connection': 'keep-alive', 'Accept-Encoding': 'gzip, deflate', 'password': '**PASSWORD**', 'Accept': '*/*', 'User-Agent': 'python-requests/2.7.0 CPython/2.7.10 Darwin/14.5.0'}
2015-12-12 22:24:24,517 INFO     [pytan.utils.external.requests.packages.urllib3.connectionpool] Starting new HTTPS connection (1): 10.0.1.240
2015-12-12 22:24:24,524 DEBUG    [pytan.utils.external.requests.packages.urllib3.connectionpool] "GET /auth HTTP/1.1" 200 134
2015-12-12 22:24:24,525 DEBUG    [pytan.session.http] HTTP response: from 'https://10.0.1.240:443/auth' len:134, status:200 OK, body type: <type 'unicode'>
2015-12-12 22:24:24,526 DEBUG    [pytan.session.http] HTTP response: headers: {'x-frame-options': 'SAMEORIGIN', 'content-type': 'text/plain; charset=us-ascii', 'connection': 'keep-alive', 'content-length': '134'}
2015-12-12 22:24:24,526 DEBUG    [pytan.session.body] HTTP response: body:
1-410-c284a982687f5d62353ae2139ccf9f416e33307769aa0a8a9e19788f9638815e15f979565a87e478ea94b98426a7d3ff442d303a5b1245f13ad43cabc3f76cdb
2015-12-12 22:24:24,526 DEBUG    [pytan.session.auth] Session ID updated to: 1-410-c284a982687f5d62353ae2139ccf9f416e33307769aa0a8a9e19788f9638815e15f979565a87e478ea94b98426a7d3ff442d303a5b1245f13ad43cabc3f76cdb
2015-12-12 22:24:24,526 DEBUG    [pytan.session.auth] Successfully authenticated and received a non-persistent (up to 5 minutes) session id using username/password
2015-12-12 22:25:10,821 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.network' to DEBUG
2015-12-12 22:25:10,822 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.extractors' to DEBUG
2015-12-12 22:25:10,822 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.mappers' to DEBUG
2015-12-12 22:25:10,822 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.parsers' to DEBUG
2015-12-12 22:25:10,822 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.pretty' to DEBUG
2015-12-12 22:25:10,822 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.tanium_obj' to DEBUG
2015-12-12 22:25:10,822 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.xml_clean' to DEBUG
2015-12-12 22:25:10,822 DEBUG    [pytan.utils.log] set_levels: setting logger 'root' to DEBUG
2015-12-12 22:26:25,746 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.network' to DEBUG
2015-12-12 22:26:25,747 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.extractors' to DEBUG
2015-12-12 22:26:25,747 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.mappers' to DEBUG
2015-12-12 22:26:25,747 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.parsers' to DEBUG
2015-12-12 22:26:25,747 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.pretty' to DEBUG
2015-12-12 22:26:25,747 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.tanium_obj' to DEBUG
2015-12-12 22:26:25,747 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.xml_clean' to DEBUG
2015-12-12 22:26:25,747 DEBUG    [pytan.utils.log] set_levels: setting logger 'root' to DEBUG
2015-12-12 22:26:31,188 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.network' to DEBUG
2015-12-12 22:26:31,188 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.extractors' to DEBUG
2015-12-12 22:26:31,189 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.mappers' to DEBUG
2015-12-12 22:26:31,189 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.parsers.parsers' to DEBUG
2015-12-12 22:26:31,189 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.pretty' to DEBUG
2015-12-12 22:26:31,189 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.tanium_obj' to DEBUG
2015-12-12 22:26:31,189 DEBUG    [pytan.utils.log] set_levels: setting logger 'pytan.utils.xml_clean' to DEBUG
2015-12-12 22:26:31,189 DEBUG    [pytan.utils.log] set_levels: setting logger 'root' to DEBUG
2015-12-12 22:55:57,786 DEBUG    [pytan.utils.network] Port test to 10.0.1.240:443 SUCCESS
2015-12-12 22:55:57,788 DEBUG    [pytan.session.auth] Using Username/Password for authentication headers
2015-12-12 22:55:57,788 DEBUG    [pytan.session.http] HTTP request: GET to https://10.0.1.240:443/auth
2015-12-12 22:55:57,788 DEBUG    [pytan.session.http] HTTP request: headers: {'username': 'QWRtaW5pc3RyYXRvcg==', 'Connection': 'keep-alive', 'Accept-Encoding': 'gzip, deflate', 'password': '**PASSWORD**', 'Accept': '*/*', 'User-Agent': 'python-requests/2.7.0 CPython/2.7.10 Darwin/14.5.0'}
2015-12-12 22:55:57,797 INFO     [pytan.utils.external.requests.packages.urllib3.connectionpool] Starting new HTTPS connection (1): 10.0.1.240
2015-12-12 22:55:57,805 DEBUG    [pytan.utils.external.requests.packages.urllib3.connectionpool] "GET /auth HTTP/1.1" 200 134
2015-12-12 22:55:57,805 DEBUG    [pytan.session.http] HTTP response: from 'https://10.0.1.240:443/auth' len:134, status:200 OK, body type: <type 'unicode'>
2015-12-12 22:55:57,805 DEBUG    [pytan.session.http] HTTP response: headers: {'x-frame-options': 'SAMEORIGIN', 'content-type': 'text/plain; charset=us-ascii', 'connection': 'keep-alive', 'content-length': '134'}
2015-12-12 22:55:57,806 DEBUG    [pytan.session.body] HTTP response: body:
1-418-a073c4335ef8394a2465777d605f12765bee436c6aaf64be6b3212750029f59c05635780af93d741827791ad3650e3e226fa51431a996dc45902f9a025386b8d
2015-12-12 22:55:57,806 DEBUG    [pytan.session.auth] Session ID updated to: 1-418-a073c4335ef8394a2465777d605f12765bee436c6aaf64be6b3212750029f59c05635780af93d741827791ad3650e3e226fa51431a996dc45902f9a025386b8d
2015-12-12 22:55:57,806 DEBUG    [pytan.session.auth] Successfully authenticated and received a non-persistent (up to 5 minutes) session id using username/password
2015-12-12 22:56:17,939 DEBUG    [pytan.utils.network] Port test to 10.0.1.240:443 SUCCESS
2015-12-12 22:56:17,939 DEBUG    [pytan.session.auth] Using Username/Password for authentication headers
2015-12-12 22:56:17,940 DEBUG    [pytan.session.http] HTTP request: GET to https://10.0.1.240:443/auth
2015-12-12 22:56:17,940 DEBUG    [pytan.session.http] HTTP request: headers: {'username': 'QWRtaW5pc3RyYXRvcg==', 'Connection': 'keep-alive', 'Accept-Encoding': 'gzip, deflate', 'password': '**PASSWORD**', 'Accept': '*/*', 'User-Agent': 'python-requests/2.7.0 CPython/2.7.10 Darwin/14.5.0'}
2015-12-12 22:56:17,949 INFO     [pytan.utils.external.requests.packages.urllib3.connectionpool] Starting new HTTPS connection (1): 10.0.1.240
2015-12-12 22:56:17,957 DEBUG    [pytan.utils.external.requests.packages.urllib3.connectionpool] "GET /auth HTTP/1.1" 200 134
2015-12-12 22:56:17,958 DEBUG    [pytan.session.http] HTTP response: from 'https://10.0.1.240:443/auth' len:134, status:200 OK, body type: <type 'unicode'>
2015-12-12 22:56:17,958 DEBUG    [pytan.session.http] HTTP response: headers: {'x-frame-options': 'SAMEORIGIN', 'content-type': 'text/plain; charset=us-ascii', 'connection': 'keep-alive', 'content-length': '134'}
2015-12-12 22:56:17,958 DEBUG    [pytan.session.body] HTTP response: body:
1-419-1fb0c7076b6435b8d83ed31c94e485482d8b0c05063841781a95ddfb82c8b338015f967550fe150a7a8301c69db87f6d4888a6a2cfdedeff04660ba4d17fdfc2
2015-12-12 22:56:17,959 DEBUG    [pytan.session.auth] Session ID updated to: 1-419-1fb0c7076b6435b8d83ed31c94e485482d8b0c05063841781a95ddfb82c8b338015f967550fe150a7a8301c69db87f6d4888a6a2cfdedeff04660ba4d17fdfc2
2015-12-12 22:56:17,959 DEBUG    [pytan.session.auth] Successfully authenticated and received a non-persistent (up to 5 minutes) session id using username/password
2015-12-12 22:57:29,746 DEBUG    [pytan.utils.network] Port test to 10.0.1.240:443 SUCCESS
2015-12-12 22:57:29,747 DEBUG    [pytan.session.auth] Using Username/Password for authentication headers
2015-12-12 22:57:29,747 DEBUG    [pytan.session.http] HTTP request: GET to https://10.0.1.240:443/auth
2015-12-12 22:57:29,747 DEBUG    [pytan.session.http] HTTP request: headers: {'username': 'QWRtaW5pc3RyYXRvcg==', 'Connection': 'keep-alive', 'Accept-Encoding': 'gzip, deflate', 'password': '**PASSWORD**', 'Accept': '*/*', 'User-Agent': 'python-requests/2.7.0 CPython/2.7.10 Darwin/14.5.0'}
2015-12-12 22:57:29,756 INFO     [pytan.utils.external.requests.packages.urllib3.connectionpool] Starting new HTTPS connection (1): 10.0.1.240
2015-12-12 22:57:29,765 DEBUG    [pytan.utils.external.requests.packages.urllib3.connectionpool] "GET /auth HTTP/1.1" 200 134
2015-12-12 22:57:29,766 DEBUG    [pytan.session.http] HTTP response: from 'https://10.0.1.240:443/auth' len:134, status:200 OK, body type: <type 'unicode'>
2015-12-12 22:57:29,766 DEBUG    [pytan.session.http] HTTP response: headers: {'x-frame-options': 'SAMEORIGIN', 'content-type': 'text/plain; charset=us-ascii', 'connection': 'keep-alive', 'content-length': '134'}
2015-12-12 22:57:29,766 DEBUG    [pytan.session.body] HTTP response: body:
1-420-09b3132d0ea2208e021dc77b2beab688fa4fbdf7e2594a7759c941bb1112fc786f526c286264ff620dea3e1c7e2f1c458f4e863b8bf7372dbb2ca21be94de447
2015-12-12 22:57:29,766 DEBUG    [pytan.session.auth] Session ID updated to: 1-420-09b3132d0ea2208e021dc77b2beab688fa4fbdf7e2594a7759c941bb1112fc786f526c286264ff620dea3e1c7e2f1c458f4e863b8bf7372dbb2ca21be94de447
2015-12-12 22:57:29,766 DEBUG    [pytan.session.auth] Successfully authenticated and received a non-persistent (up to 5 minutes) session id using username/password
2015-12-12 23:00:54,834 DEBUG    [pytan.utils.log] add_handler: pytan.log to logger pytan.utils.network
2015-12-12 23:00:54,835 DEBUG    [pytan.utils.log] add_handler: pytan.log to logger pytan.utils.parsers.extractors
2015-12-12 23:00:54,835 DEBUG    [pytan.utils.log] add_handler: pytan.log to logger pytan.utils.parsers.mappers
2015-12-12 23:00:54,835 DEBUG    [pytan.utils.log] add_handler: pytan.log to logger pytan.utils.parsers.parsers
2015-12-12 23:00:54,835 DEBUG    [pytan.utils.log] add_handler: pytan.log to logger pytan.utils.pretty
2015-12-12 23:00:54,835 DEBUG    [pytan.utils.log] add_handler: pytan.log to logger pytan.utils.tanium_obj
2015-12-12 23:00:54,835 DEBUG    [pytan.utils.log] add_handler: pytan.log to logger pytan.utils.xml_clean
2015-12-12 23:00:54,835 DEBUG    [pytan.utils.log] added file log name: 'pytan.log', output: 'pytan.log'
2015-12-12 23:00:54,836 DEBUG    [pytan.utils.network] Port test to 10.0.1.240:443 SUCCESS
2015-12-12 23:00:54,837 DEBUG    [pytan.session.auth] Using Username/Password for authentication headers
2015-12-12 23:00:54,837 DEBUG    [pytan.session.http] HTTP request: GET to https://10.0.1.240:443/auth
2015-12-12 23:00:54,837 DEBUG    [pytan.session.http] HTTP request: headers: {'username': 'QWRtaW5pc3RyYXRvcg==', 'Connection': 'keep-alive', 'Accept-Encoding': 'gzip, deflate', 'password': '**PASSWORD**', 'Accept': '*/*', 'User-Agent': 'python-requests/2.7.0 CPython/2.7.10 Darwin/14.5.0'}
2015-12-12 23:00:54,846 INFO     [pytan.utils.external.requests.packages.urllib3.connectionpool] Starting new HTTPS connection (1): 10.0.1.240
2015-12-12 23:00:54,853 DEBUG    [pytan.utils.external.requests.packages.urllib3.connectionpool] "GET /auth HTTP/1.1" 200 134
2015-12-12 23:00:54,853 DEBUG    [pytan.session.http] HTTP response: from 'https://10.0.1.240:443/auth' len:134, status:200 OK, body type: <type 'unicode'>
2015-12-12 23:00:54,853 DEBUG    [pytan.session.http] HTTP response: headers: {'x-frame-options': 'SAMEORIGIN', 'content-type': 'text/plain; charset=us-ascii', 'connection': 'keep-alive', 'content-length': '134'}
2015-12-12 23:00:54,854 DEBUG    [pytan.session.body] HTTP response: body:
1-424-6e0537fba079266dec5c7f0b77a3743195cc199c599caf47239445cca9929f40723f838b3fe25e8fc4523c85212109244f3bfb2ffb2aa10897f70714862c7743
2015-12-12 23:00:54,854 DEBUG    [pytan.session.auth] Session ID updated to: 1-424-6e0537fba079266dec5c7f0b77a3743195cc199c599caf47239445cca9929f40723f838b3fe25e8fc4523c85212109244f3bfb2ffb2aa10897f70714862c7743
2015-12-12 23:00:54,854 DEBUG    [pytan.session.auth] Successfully authenticated and received a non-persistent (up to 5 minutes) session id using username/password
2015-12-12 23:13:45,310 DEBUG    [pytan.utils.log] added file log name: 'pytan.log', output: 'pytan.log'
2015-12-12 23:13:45,312 DEBUG    [pytan.utils.network] Port test to 10.0.1.240:443 SUCCESS
2015-12-12 23:13:45,312 DEBUG    [pytan.session.auth] Using Username/Password for authentication headers
2015-12-12 23:13:45,312 DEBUG    [pytan.session.http] HTTP request: GET to https://10.0.1.240:443/auth
2015-12-12 23:13:45,312 DEBUG    [pytan.session.http] HTTP request: headers: {'username': 'QWRtaW5pc3RyYXRvcg==', 'Connection': 'keep-alive', 'Accept-Encoding': 'gzip, deflate', 'password': '**PASSWORD**', 'Accept': '*/*', 'User-Agent': 'python-requests/2.7.0 CPython/2.7.10 Darwin/14.5.0'}
2015-12-12 23:13:45,321 INFO     [pytan.utils.external.requests.packages.urllib3.connectionpool] Starting new HTTPS connection (1): 10.0.1.240
2015-12-12 23:13:45,330 DEBUG    [pytan.utils.external.requests.packages.urllib3.connectionpool] "GET /auth HTTP/1.1" 200 134
2015-12-12 23:13:45,331 DEBUG    [pytan.session.http] HTTP response: from 'https://10.0.1.240:443/auth' len:134, status:200 OK, body type: <type 'unicode'>
2015-12-12 23:13:45,331 DEBUG    [pytan.session.http] HTTP response: headers: {'x-frame-options': 'SAMEORIGIN', 'content-type': 'text/plain; charset=us-ascii', 'connection': 'keep-alive', 'content-length': '134'}
2015-12-12 23:13:45,331 DEBUG    [pytan.session.body] HTTP response: body:
1-425-e8245c01199b4155ea1bba996ccb9f1dd1fe4b6248bf6cdc7b5d94868f27e239ce0e445b10bd6df2a5d11a888e8c74e6590071e98fdc286f2bd2384b729976fc
2015-12-12 23:13:45,331 DEBUG    [pytan.session.auth] Session ID updated to: 1-425-e8245c01199b4155ea1bba996ccb9f1dd1fe4b6248bf6cdc7b5d94868f27e239ce0e445b10bd6df2a5d11a888e8c74e6590071e98fdc286f2bd2384b729976fc
2015-12-12 23:13:45,331 DEBUG    [pytan.session.auth] Successfully authenticated and received a non-persistent (up to 5 minutes) session id using username/password
2015-12-12 23:14:07,294 DEBUG    [pytan.utils.log] added file log name: 'pytan.log', output: 'pytan.log'
2015-12-12 23:14:07,296 DEBUG    [pytan.utils.network] Port test to 10.0.1.240:443 SUCCESS
2015-12-12 23:14:07,297 DEBUG    [pytan.session.auth] Using Username/Password for authentication headers
2015-12-12 23:14:07,297 DEBUG    [pytan.session.http] HTTP request: GET to https://10.0.1.240:443/auth
2015-12-12 23:14:07,297 DEBUG    [pytan.session.http] HTTP request: headers: {'username': 'QWRtaW5pc3RyYXRvcg==', 'Connection': 'keep-alive', 'Accept-Encoding': 'gzip, deflate', 'password': '**PASSWORD**', 'Accept': '*/*', 'User-Agent': 'python-requests/2.7.0 CPython/2.7.10 Darwin/14.5.0'}
2015-12-12 23:14:07,306 INFO     [pytan.utils.external.requests.packages.urllib3.connectionpool] Starting new HTTPS connection (1): 10.0.1.240
2015-12-12 23:14:07,316 DEBUG    [pytan.session.http] HTTP response: from 'https://10.0.1.240:443/auth' len:134, status:200 OK, body type: <type 'unicode'>
2015-12-12 23:14:07,316 DEBUG    [pytan.session.http] HTTP response: headers: {'x-frame-options': 'SAMEORIGIN', 'content-type': 'text/plain; charset=us-ascii', 'connection': 'keep-alive', 'content-length': '134'}
2015-12-12 23:14:07,316 DEBUG    [pytan.session.body] HTTP response: body:
1-427-c932ea8a33a3d08b4988789dacb7a4eb8d79208542f50d097b05ac4b44d13d4982e57c6467441b438abab65f83005ab992bc7eb706fc93cefc0b9d32f1b96f03
2015-12-12 23:14:07,316 DEBUG    [pytan.session.auth] Session ID updated to: 1-427-c932ea8a33a3d08b4988789dacb7a4eb8d79208542f50d097b05ac4b44d13d4982e57c6467441b438abab65f83005ab992bc7eb706fc93cefc0b9d32f1b96f03
2015-12-12 23:14:07,316 DEBUG    [pytan.session.auth] Successfully authenticated and received a non-persistent (up to 5 minutes) session id using username/password
2015-12-12 23:16:36,491 DEBUG    [pytan.utils.log] added file log name: 'pytan.log', output: 'pytan.log'
2015-12-12 23:16:36,493 DEBUG    [pytan.utils.network] Port test to 10.0.1.240:443 SUCCESS
2015-12-12 23:16:36,493 DEBUG    [pytan.session.auth] Using Username/Password for authentication headers
2015-12-12 23:16:36,493 DEBUG    [pytan.session.http] HTTP request: GET to https://10.0.1.240:443/auth
2015-12-12 23:16:36,493 DEBUG    [pytan.session.http] HTTP request: headers: {'username': 'QWRtaW5pc3RyYXRvcg==', 'Connection': 'keep-alive', 'Accept-Encoding': 'gzip, deflate', 'password': '**PASSWORD**', 'Accept': '*/*', 'User-Agent': 'python-requests/2.7.0 CPython/2.7.10 Darwin/14.5.0'}
2015-12-12 23:16:36,503 INFO     [pytan.utils.external.requests.packages.urllib3.connectionpool] Starting new HTTPS connection (1): 10.0.1.240
2015-12-12 23:16:36,509 DEBUG    [pytan.session.http] HTTP response: from 'https://10.0.1.240:443/auth' len:134, status:200 OK, body type: <type 'unicode'>
2015-12-12 23:16:36,509 DEBUG    [pytan.session.http] HTTP response: headers: {'x-frame-options': 'SAMEORIGIN', 'content-type': 'text/plain; charset=us-ascii', 'connection': 'keep-alive', 'content-length': '134'}
2015-12-12 23:16:36,509 DEBUG    [pytan.session.body] HTTP response: body:
1-428-1d2da2e3e266ac365f43001342f12364e586d8c031727e05dfa1947f543a7cf3d38c4071547510cff2810ee4800234d8d74f6ef9f5d0fb3cfa6aca284899f09e
2015-12-12 23:16:36,510 DEBUG    [pytan.session.auth] Session ID updated to: 1-428-1d2da2e3e266ac365f43001342f12364e586d8c031727e05dfa1947f543a7cf3d38c4071547510cff2810ee4800234d8d74f6ef9f5d0fb3cfa6aca284899f09e
2015-12-12 23:16:36,510 DEBUG    [pytan.session.auth] Successfully authenticated and received a non-persistent (up to 5 minutes) session id using username/password
2015-12-12 23:16:45,770 DEBUG    [pytan.utils.network] Port test to 10.0.1.240:443 SUCCESS
2015-12-12 23:16:45,771 DEBUG    [pytan.session.auth] Using Username/Password for authentication headers
2015-12-12 23:16:45,771 DEBUG    [pytan.session.http] HTTP request: GET to https://10.0.1.240:443/auth
2015-12-12 23:16:45,771 DEBUG    [pytan.session.http] HTTP request: headers: {'username': 'QWRtaW5pc3RyYXRvcg==', 'Connection': 'keep-alive', 'Accept-Encoding': 'gzip, deflate', 'password': '**PASSWORD**', 'Accept': '*/*', 'User-Agent': 'python-requests/2.7.0 CPython/2.7.10 Darwin/14.5.0'}
2015-12-12 23:16:45,781 INFO     [pytan.utils.external.requests.packages.urllib3.connectionpool] Starting new HTTPS connection (1): 10.0.1.240
2015-12-12 23:16:45,790 DEBUG    [pytan.session.http] HTTP response: from 'https://10.0.1.240:443/auth' len:134, status:200 OK, body type: <type 'unicode'>
2015-12-12 23:16:45,790 DEBUG    [pytan.session.http] HTTP response: headers: {'x-frame-options': 'SAMEORIGIN', 'content-type': 'text/plain; charset=us-ascii', 'connection': 'keep-alive', 'content-length': '134'}
2015-12-12 23:16:45,790 DEBUG    [pytan.session.body] HTTP response: body:
1-429-72cb08c87cc303065d583b70f53be7ee520e29ff52335b4b4d5c1c0dd76377ddc96588eb12acb1537072c4da78d4ed982cb9a8fd73d92328622d5774f7d76d5e
2015-12-12 23:16:45,790 DEBUG    [pytan.session.auth] Session ID updated to: 1-429-72cb08c87cc303065d583b70f53be7ee520e29ff52335b4b4d5c1c0dd76377ddc96588eb12acb1537072c4da78d4ed982cb9a8fd73d92328622d5774f7d76d5e
2015-12-12 23:16:45,790 DEBUG    [pytan.session.auth] Successfully authenticated and received a non-persistent (up to 5 minutes) session id using username/password
2015-12-12 23:18:22,612 DEBUG    [pytan.utils.log] added file log name: 'pytan.log', output: 'pytan.log'
2015-12-12 23:18:22,614 DEBUG    [pytan.utils.network] Port test to 10.0.1.240:443 SUCCESS
2015-12-12 23:18:22,614 DEBUG    [pytan.session.auth] Using Username/Password for authentication headers
2015-12-12 23:18:22,614 DEBUG    [pytan.session.http] HTTP request: GET to https://10.0.1.240:443/auth
2015-12-12 23:18:22,614 DEBUG    [pytan.session.http] HTTP request: headers: {'username': 'QWRtaW5pc3RyYXRvcg==', 'Connection': 'keep-alive', 'Accept-Encoding': 'gzip, deflate', 'password': '**PASSWORD**', 'Accept': '*/*', 'User-Agent': 'python-requests/2.7.0 CPython/2.7.10 Darwin/14.5.0'}
2015-12-12 23:18:22,623 INFO     [pytan.utils.external.requests.packages.urllib3.connectionpool] Starting new HTTPS connection (1): 10.0.1.240
2015-12-12 23:18:22,633 DEBUG    [pytan.session.http] HTTP response: from 'https://10.0.1.240:443/auth' len:134, status:200 OK, body type: <type 'unicode'>
2015-12-12 23:18:22,633 DEBUG    [pytan.session.http] HTTP response: headers: {'x-frame-options': 'SAMEORIGIN', 'content-type': 'text/plain; charset=us-ascii', 'connection': 'keep-alive', 'content-length': '134'}
2015-12-12 23:18:22,633 DEBUG    [pytan.session.body] HTTP response: body:
1-430-cf3fe7a87adb568f375a9da52cc4eff94a6b1331c54d5491b3e5282989e08a8f01abde85f29919c52fe84b3615c08d82a5fb92b87afa7436c2ae61fc800d3ff1
2015-12-12 23:18:22,634 DEBUG    [pytan.session.auth] Session ID updated to: 1-430-cf3fe7a87adb568f375a9da52cc4eff94a6b1331c54d5491b3e5282989e08a8f01abde85f29919c52fe84b3615c08d82a5fb92b87afa7436c2ae61fc800d3ff1
2015-12-12 23:18:22,634 DEBUG    [pytan.session.auth] Successfully authenticated and received a non-persistent (up to 5 minutes) session id using username/password
2015-12-12 23:18:25,714 DEBUG    [pytan.utils.log] added file log name: 'pytan.log', output: 'pytan.log'
2015-12-12 23:18:25,716 DEBUG    [pytan.utils.network] Port test to 10.0.1.240:443 SUCCESS
2015-12-12 23:18:25,717 DEBUG    [pytan.session.auth] Using Username/Password for authentication headers
2015-12-12 23:18:25,717 DEBUG    [pytan.session.http] HTTP request: GET to https://10.0.1.240:443/auth
2015-12-12 23:18:25,717 DEBUG    [pytan.session.http] HTTP request: headers: {'username': 'QWRtaW5pc3RyYXRvcg==', 'Connection': 'keep-alive', 'Accept-Encoding': 'gzip, deflate', 'password': '**PASSWORD**', 'Accept': '*/*', 'User-Agent': 'python-requests/2.7.0 CPython/2.7.10 Darwin/14.5.0'}
2015-12-12 23:18:25,726 INFO     [pytan.utils.external.requests.packages.urllib3.connectionpool] Starting new HTTPS connection (1): 10.0.1.240
2015-12-12 23:18:25,733 DEBUG    [pytan.utils.external.requests.packages.urllib3.connectionpool] "GET /auth HTTP/1.1" 200 134
2015-12-12 23:18:25,734 DEBUG    [pytan.session.http] HTTP response: from 'https://10.0.1.240:443/auth' len:134, status:200 OK, body type: <type 'unicode'>
2015-12-12 23:18:25,734 DEBUG    [pytan.session.http] HTTP response: headers: {'x-frame-options': 'SAMEORIGIN', 'content-type': 'text/plain; charset=us-ascii', 'connection': 'keep-alive', 'content-length': '134'}
2015-12-12 23:18:25,734 DEBUG    [pytan.session.body] HTTP response: body:
1-431-b86d458638c3cd9bbee5b421e6603e439afb307f2f9a7e1bd301c0c442c2302093d27265157491b9d270a3e71c238df63f15f25f96ffc7e316a0b30341035431
2015-12-12 23:18:25,734 DEBUG    [pytan.session.auth] Session ID updated to: 1-431-b86d458638c3cd9bbee5b421e6603e439afb307f2f9a7e1bd301c0c442c2302093d27265157491b9d270a3e71c238df63f15f25f96ffc7e316a0b30341035431
2015-12-12 23:18:25,734 DEBUG    [pytan.session.auth] Successfully authenticated and received a non-persistent (up to 5 minutes) session id using username/password
2015-12-12 23:18:39,059 DEBUG    [pytan.utils.log] added file log name: 'pytan.log', output: 'pytan.log'
2015-12-12 23:18:39,060 DEBUG    [pytan.utils.network] Port test to 10.0.1.240:443 SUCCESS
2015-12-12 23:18:39,061 DEBUG    [pytan.session.auth] Using Username/Password for authentication headers
2015-12-12 23:18:39,061 DEBUG    [pytan.session.http] HTTP request: GET to https://10.0.1.240:443/auth
2015-12-12 23:18:39,061 DEBUG    [pytan.session.http] HTTP request: headers: {'username': 'QWRtaW5pc3RyYXRvcg==', 'Connection': 'keep-alive', 'Accept-Encoding': 'gzip, deflate', 'password': '**PASSWORD**', 'Accept': '*/*', 'User-Agent': 'python-requests/2.7.0 CPython/2.7.10 Darwin/14.5.0'}
2015-12-12 23:18:39,070 INFO     [pytan.utils.external.requests.packages.urllib3.connectionpool] Starting new HTTPS connection (1): 10.0.1.240
2015-12-12 23:18:39,077 DEBUG    [pytan.session.http] HTTP response: from 'https://10.0.1.240:443/auth' len:134, status:200 OK, body type: <type 'unicode'>
2015-12-12 23:18:39,077 DEBUG    [pytan.session.http] HTTP response: headers: {'x-frame-options': 'SAMEORIGIN', 'content-type': 'text/plain; charset=us-ascii', 'connection': 'keep-alive', 'content-length': '134'}
2015-12-12 23:18:39,078 DEBUG    [pytan.session.body] HTTP response: body:
1-432-b16c5d5a836d1e69abdac023b06da843a5851926a0a8d3add67d77ced2938d5817836a2077a723efd586ed40345ec6d820212a2db8d3752587316bc54db66072
2015-12-12 23:18:39,078 DEBUG    [pytan.session.auth] Session ID updated to: 1-432-b16c5d5a836d1e69abdac023b06da843a5851926a0a8d3add67d77ced2938d5817836a2077a723efd586ed40345ec6d820212a2db8d3752587316bc54db66072
2015-12-12 23:18:39,078 DEBUG    [pytan.session.auth] Successfully authenticated and received a non-persistent (up to 5 minutes) session id using username/password
2015-12-12 23:19:57,634 DEBUG    [pytan.utils.log] added file log name: 'pytan.log', output: 'pytan.log'
2015-12-12 23:19:57,636 DEBUG    [pytan.utils.network] Port test to 10.0.1.240:443 SUCCESS
2015-12-12 23:19:57,637 DEBUG    [pytan.session.auth] Using Username/Password for authentication headers
2015-12-12 23:19:57,637 DEBUG    [pytan.session.http] HTTP request: GET to https://10.0.1.240:443/auth
2015-12-12 23:19:57,637 DEBUG    [pytan.session.http] HTTP request: headers: {'username': 'QWRtaW5pc3RyYXRvcg==', 'Connection': 'keep-alive', 'Accept-Encoding': 'gzip, deflate', 'password': '**PASSWORD**', 'Accept': '*/*', 'User-Agent': 'python-requests/2.7.0 CPython/2.7.10 Darwin/14.5.0'}
2015-12-12 23:19:57,646 INFO     [pytan.utils.external.requests.packages.urllib3.connectionpool] Starting new HTTPS connection (1): 10.0.1.240
2015-12-12 23:19:57,654 DEBUG    [pytan.session.http] HTTP response: from 'https://10.0.1.240:443/auth' len:134, status:200 OK, body type: <type 'unicode'>
2015-12-12 23:19:57,654 DEBUG    [pytan.session.http] HTTP response: headers: {'x-frame-options': 'SAMEORIGIN', 'content-type': 'text/plain; charset=us-ascii', 'connection': 'keep-alive', 'content-length': '134'}
2015-12-12 23:19:57,654 DEBUG    [pytan.session.body] HTTP response: body:
1-433-4c336ec21169d3427bdf5fb8e2f0b7b773a19ed3f215b46969887a0732f3caad3d1c2d90e844a3447f7bd0f7e38c3fee4c19e73a3856f3decff9fbfa362ac7c1
2015-12-12 23:19:57,654 DEBUG    [pytan.session.auth] Session ID updated to: 1-433-4c336ec21169d3427bdf5fb8e2f0b7b773a19ed3f215b46969887a0732f3caad3d1c2d90e844a3447f7bd0f7e38c3fee4c19e73a3856f3decff9fbfa362ac7c1
2015-12-12 23:19:57,655 DEBUG    [pytan.session.auth] Successfully authenticated and received a non-persistent (up to 5 minutes) session id using username/password
